cdap-ui/app/cdap/services/global-constants.js

```diff
@@ -23,7 +23,7 @@ const pluginLabels = {
   'action': 'Conditions and Actions',
   'erroralert': 'Error Handlers and Alerts'
 };
-export const NUMBER_TYPES = ['integer', 'short', 'long', 'float', 'double'];
+export const NUMBER_TYPES = ['integer', 'int', 'short', 'long', 'float', 'double'];
```

Comment:
there's integer and int?

Reply:
Apparently I was seeing both the types mixed up somehow. Will finalize with both Albert and nitin and will remove whichever is un-necessary.